### ISSUE TITLE
Error loading subscribtion videos and comments

### ISSUE THREAD

**Issue by Lako29** [FreeTubeApp/FreeTube#1021]
**Behavior of the program**
When I click on the subscriptions button on the side bar, I get the following error: 
TypeError: Cannot read property 'title' of undefined
It subsequently shows several of these error messages, and then loads the videos.

When I try to open comments, I get:
Error: Request failed with status code 404
It subsequently shows several of these error messages, then says 'falling back on Invidious API', and then sometimes loads the comments, sometimes not. 

**Expected behavior**
I would like the videos of subscriptions to load without error, and for comments to be available without errors.

**To Reproduce**
Please add all steps to reproduce the behavior:
1. Go to 'subscriptions'.
2. See error
1. Go to comments of any video.
2. See error

**Environment Information (please complete the following information):**
 - Windows 10
 - FreeTube version:  0.11.2
 - Installation Method (If applicable): exe
 - Primary API Used: Local API

**Additional context**
Note: non-technical user here.

**Comment by saideepesh000** [comment]
@Lako29, In settings, try using another instance of the Invidious API and keep on changing that this.

**Comment by Lako29** [comment]
It works :) Thanks!

**Comment by Mennaruuk** [comment]
Hi! I still get the same batch of errors regardless of selected Invidious instance:
![image](https://user-images.githubusercontent.com/52135169/107598669-738ce700-6bec-11eb-9c46-b044c00b94a7.png)

**Comment by Janaue** [comment]
same

**Comment by JLP04** [comment]
Same

**Comment by Lako29** [comment]
Yes me too now. It worked for a while but the 'solution' is very unstable and widely varies on any given time.

**Comment by a20eac1d** [comment]
Yes, I have the same issue.

**Comment by fionamorrison13** [comment]
Same issue here.

**Comment by esotericDisciple** [comment]
Same, and channels only show one page of videos, [an issue NewPipe is also having...](https://github.com/TeamNewPipe/NewPipe/issues/5550)

**Comment by PrestonN** [comment]
Fix is coming out this weekend. There's a problem with the module and I already have a WIP for the fix. I'll try to get this fixed as soon as I can. For now, having fallback to Invidious enabled seems to still be working so that should be a workaround for now. For subscriptions enabling RSS for your feed should avoid seeing all the errors showing up.

**Comment by gorrioncillo** [comment]
same here, I try to change the invidious works, more o less, but the time's videos are wrong on subscription section,  but when using the button of the particular channel it works the correct time of the videos.

**Comment by GilgusMaximus** [comment]
Closing because this has been fixed in the last update

**Comment by via80HD** [comment]
I have been having this problem with the _"local error 404" "falling back to invidious API"_ and the _"type error cannot fetch thumbnail"_ error but have only been able to get a screenshot showing the _"local error 404" "falling back to invidious API"_ error. I have tried quite a few different instances both invidious and other instances from the list. I also deleted all of my subscriptions and deleted the app then re-downloaded it just to see if it was a banned channel or something and it still does it though it does seem to do it less often? Any idea of what I can do to fix it?

Thank you
